modules/ROOT/pages/enable-inbound-traffic.adoc

```diff
@@ -3,208 +3,125 @@ ifndef::env-site,env-github[]
+All inbound traffic entering Anypoint Runtime Fabric is encrypted using TLS. When enabling inbound traffic, you must provide a valid TLS certificate. Once enabled, HTTP requests sent to Runtime Fabric receive a `301` response to redirect to HTTPS on port 443.
 +
-[NOTE]
-If you do not see *Add TLS Context*, ensure you are viewing your secret group in edit mode. Return to the *Secret Groups* page and then select *Edit* next to your group.
-
-.. Type a name for your TLS context. This will identify it when you apply it to your Runtime Fabric later.
-.. Use the default TLS version (TLS 1.2).
-.. Under *Target*, select *Anypoint Security*. This configures your TLS context so it can be applied to a Runtime Fabric.
-.. Under *Keystore*, select the name of the keystore you configured previously.
-.. Select *Save*.
-
-. After saving your TLS context, select *Secret Groups* and then select *Finish*. This enables TLS Context for your *Runtime Fabric.*
+You can configure the private key and public certificate for a TLS-enabled server in one of the following ways:
 
-== Enable Inbound Traffic
-The Runtime Fabric internal load balancer supports OpenSSL 1.1.1 and TLS 1.3, which provide the following security enhancements and performance improvements over TLS 1.2:
- 
-* 2x or greater TLS 1.2 connection performance throughput boost versus prior Runtime Fabric internal load balancer running on OpenSSL 1.0.2
-* Reduction of one round trip in full handshake for TLS 1.3 vs. TLS 1.2
-* Added TLS 1.2 and TLS 1.3 ChaCha20-Poly1305 ciphers for better mobile and IoT device support
-* TLS 1.3 protection against downgrade attacks
-* Continued support for both RSA and ECDSA keys
-* TLS 1.3 removes obsolete and insecure features from TLS 1.2 and TLS 1.1. Refer to the xref:release-notes::anypoint-security/anypoint-security-rtf-edge.adoc[Anypoint Security Edge Release Notes] for a complete list of supported and deprecated ciphers. 
+** Option 1: *Upload PEM*
++
+Use this option to upload a public certificate and private key in Privacy-Enhanced Mail (PEM) format and use TLS default values. With this option, you cannot change the default values for TLS versions, ciphers, and other TLS configuration options. A PEM file is a Base64-encoded ASCII file with a `.cer`, `.crt`, or `.pem` extension. Refer to xref:anypoint-security::configure-tls-context-rtf-lb.adoc[Configure a TLS Context for Runtime Fabric Load Balancer] for  information about TLS default values.
++
+This is the default option when no TLS context exists for Runtime Fabric.
 
-The TLS 1.3 protocol is enabled by default for those clients that support it.
+... Specify a *Certificate File*. When you specify a PEM public certificate file, the *Derived URL Format* field lists domains that are selectable for *Application url*, which is used for routing requests to an application. By default, the first domain listed is used. Other values can be selected via the *Applications->Ingress* page.
```

Comment:
GLOBALLY restrict steps and other list items to one fragment or sentence max, with additional details, if necessary, in a para following:
```suggestion
... Specify a *Certificate File*. 
When you specify a PEM public certificate file, the *Derived URL Format* field lists domains that are selectable for *Application url*, which is used for routing requests to an application. By default, the first domain listed is used. Other values can be selected via the *Applications->Ingress* page.
```